velodyne_msgs/CMakeLists.txt

```diff
@@ -1,16 +1,24 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
+
 project(velodyne_msgs)
 
-find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-add_message_files(
-  DIRECTORY msg
-  FILES
-  VelodynePacket.msg
-  VelodyneScan.msg
-)
-generate_messages(DEPENDENCIES std_msgs)
+find_package(tinyxml2 REQUIRED)
```

Comment:
This shouldn't be required.

Reply:
Something on my local mac that requires me to have it, but will keep it local and remove it from the repo.